src/tiled/commandlineparser.cpp

```diff
@@ -118,13 +118,11 @@ bool CommandLineParser::parse(const QStringList &arguments)
 
 void CommandLineParser::showHelp()
 {
-    // TODO: Make translatable
-    qWarning().nospace() << "Usage:\n"
-                         << "  " << qPrintable(mCurrentProgramName)
-                         << " [options] [files...]\n\n"
-                         << "Options:";
+    qWarning().nospace() << tr("Usage:\n  ") << qPrintable(mCurrentProgramName)
+                         << tr(" [options] [files...]\n\n")
```

Comment:
Please do not include `\n\n` at the end of the translatable string, since they're easy to miss for translators. For those it is really better to have them outside of the translatable part.